### DISCUSSION TITLE
Include images in messages to attendees and orders contacts

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#212] [Ideas]
I was looking at the messages option, and realised that there isn't a way to include any images when creating the content. Is this something that can be added as a feature? We often embed a few images that relate to the event. The messaging feature is a great  resource to have. Well done

**Comment by daveearley** [comment]
There as no immediate plans to add support for adding images to messages. But I wouldn't rule it out in the future.

**Reply by justynpride** [reply]
Thanks for the helpful feedback. Would an alternative be to allow the edit of the html as some editors allow? Admittedly this is only helpful for those who know how to do the code.
